snuba/reader.py

```diff
@@ -70,16 +70,18 @@ def iterate_rows():
     for col in result["meta"]:
         if DATETIME_TYPE_RE.match(col["type"]):
             for row in iterate_rows():
-                row[col["name"]] = (
-                    row[col["name"]].replace(tzinfo=tz.tzutc()).isoformat()
-                )
+                if row[col["name"]] is not None:  # The column value can be null/None at times.
```

Comment:
This is a valid condition only if the regex match above contains "Nullable" otherwise, this being null, would be an error. I think we should add that check